mitxgraders/helpers/calc.py

```diff
@@ -164,7 +167,11 @@ def get_parser(self, formula, suffixes):
 # The global parser cache
 parsercache = ParserCache()
 
-def evaluator(formula, variables, functions, suffixes, allow_vectors=False):
+def evaluator(formula,
+              variables=DEFAULT_VARIABLES,
```

Comment:
Just confirming: You don't mind the mutable default argument because nothing mutates this below? 

Otherwise, 👍 So if you're fine with it, feel free to merge.

Reply:
I'm fine with it. An alternative is to have them set to `NONE`, check that and then default to the variables, but given that nothing mutates it anyway, I don't think anything is going to go wrong.

Reply:
And this is just a nicer implementation, too...